app/poros/github/api.rb

```diff
@@ -0,0 +1,25 @@
+require 'faraday'
+require 'json'
+require_relative 'follower'
+require_relative 'repo'
+
+class Api
+  attr_reader :followers, :repos
+
+  def initialize(current_user)
+    @current_user = current_user
+    @connection = connect
+    @followers = parse_info('followers').map { |follower| Follower.new(follower) }
```

Comment:
This could be broken out into a helper method, just kind of curious when/where you'd call it. Being able to call `@response.followers.each` and each `follower` being a `Follower class` seems pretty clean.